docs/marionette.functions.md

```diff
@@ -15,10 +15,9 @@ a way to get the same behaviors and conventions from your own code.
 * [Marionette.mergeOptions](#marionettemergeoptions)
 * [Marionette.getOption](#marionettegetoption)
 * [Marionette.triggerMethod](#marionettetriggermethod)
+* [Marionette.triggerMethodOn](#marionettetriggermethodon)
 * [Marionette.bindEvents](#marionettebindevents)
 * [Marionette.bindRequests](#marionettebindrequests)
-* [Marionette.unbindRequests](#marionetteunbindrequests)
```

Comment:
Oh I think it is ok to leave it in the options..
just that the order be:
bindevents
unbindevents
bindrequests
unbindrequests